manifests/gcp_marketplace/chart/kubeflow-pipelines/templates/pipeline.yaml

```diff
@@ -608,6 +608,7 @@ spec:
           name: ml-pipeline-visualizationserver
           ports:
             - containerPort: 8888
+      serviceAccountName: ml-pipeline-visualizationserver
```

Comment:
You also need to add the service account yaml for hosted.

Reply:
done